demos/android/MASVS-CRYPTO/MASTG-DEMO-0015/MastgTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.owasp.mastestapp
```

Comment:
Please test this code using https://github.com/cpholguera/MASTestApp-Android 

Please follow all instructions in the README.

You can use the included script to generate the reversed java file: https://github.com/cpholguera/MASTestApp-Android?tab=readme-ov-file#run-the-extraction-script

We only need `MastgTest_reversed.java` in this case so you can ignore the manifest.